ol8/profiles/ospp.profile

```diff
@@ -0,0 +1,207 @@
+documentation_complete: true
+
+title: 'OSPP - Protection Profile for General Purpose Operating Systems'
+
+description: |-
+    This profile reflects mandatory configuration controls identified in the
+    NIAP Configuration Annex to the Protection Profile for General Purpose
+    Operating Systems (Protection Profile Version 4.2).
```

Comment:
@iokomin I would add the following to the description:

```
This profile is currently under review. Use of this profile does not denote or guarantee NIAP approval or certification
until this profile has been approved by NIAP.
```
Then when OL is NIAP validated and certited, remove the above sentences.

Reply:
@redhatrises please find draft note added to profile description.